### PR TITLE
Project One | Basic page structure.

### PR DESCRIPTION
**Pull request by errea**
># Description

The activity I was able to set up a JavaScript project for the Leaderboard list app, using Webpack and ES6 features, notably modules, and this PR is to check for the basic functionality of the project setup.

># Demo
* For each flow, post a video or gif. `Brief description of image`

Screen Shot 2021-07-27 at 10 30 05 PM

# How Has This Been Tested?

No test for this project at this stage, since this is the setup stages of the project

- [ ] Jest

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My code flows gitflow 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] My code uses Webpack to compile JavaScript modules.
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,227 @@
...
+    "word-wrap": "^1.2.3",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "zwitch": "^1.0.5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
```
Comment (abruzy) [inline package.json]: Well done implementing this but you want to add the mode for development and production in your script like this below 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <h2>Add your score</h2>
+        <form id="add-form" method="POST">
+          <input type="text" placeholder="Your name" />
+          <input type="text" placeholder="Your score" />
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+    </main>
+  </body>
+</html>
```
Comment (abruzy) [inline src/index.html:35]: I can see that you have the same content in the two `index.html` files you have

Screenshot 2021-07-27 at 23 09 07

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @errea 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: package.json (hunk 1 above)
Comment (errea) [reply package.json]: ok @abruzy thank you for the review

**Review by abruzy — APPROVED** [review]
```markdown
Hi @errea 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**